### PR TITLE
fix: clean up overlay a bit more

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

**Comment by jquense** [comment]
@taion what do you think about this for a render prop format generally, anything that is meant to be spread through is on `props`.

File: src/Dropdown.js [hunk 1]
```diff
@@ -202,7 +224,7 @@ class Dropdown extends React.Component {
     return (
       <DropdownContext.Provider value={this.state.context}>
         <Popper.Manager>
-          {children({ onKeyDown: this.handleKeyDown })}
+          {children({ props: { onKeyDown: this.handleKeyDown } })}
```
Comment (taion) [inline src/Dropdown.js]: I think I get what you're trying to do here, but I'm a little confused by this API.

For example, does it really make sense for `onKeyDown` here to be under `props`, but for `onToggle` below to not be thus?

Comment (jquense) [reply src/Dropdown.js]: My thought was to try and make crystal clear what items to spread to a host element and have a place to add additional arguments withough breaking existing components that are spreading. We should change on toggle to just toggle maybe if that's clearer?

Comment (taion) [reply src/Dropdown.js]: it's weird that we force you into this `onKeyDown` thing without, say, doing the same for `onClick` on toggle :p

Comment (taion) [reply src/Dropdown.js]: but i guess renaming it from `onToggle` to `toggle` would work!

**Pull request merged**